data/pidgin/cpsAssets/23248703.json

```diff
@@ -1,334 +1,387 @@
 {
     "metadata": {
-        "id": "urn:bbc:ares::asset:pidgin/23248703",
-        "locators": {
-            "assetUri": "/pidgin/23248703",
-            "cpsUrn": "urn:bbc:content:assetUri:pidgin/23248703",
-            "curie": "http://www.bbc.co.uk/asset/5679389a-3ea6-0b40-9de4-f4d33d6bcd9f"
+      "id": "urn:bbc:ares::asset:pidgin/23248703",
```

Comment:
This is a copy paste of the latest version of this asset from Ares, the main changes being a headline containing `\"` and the first paragraph having `&quot;` and `'` to give example of double and single quotes working as expected.